spring-cloud-dataflow-server-local/src/main/resources/META-INF/kafka-apps.properties

```diff
@@ -37,3 +37,8 @@ sink.tcp=maven://org.springframework.cloud.stream.app:tcp-sink-kafka:@registry.a
 sink.throughput=maven://org.springframework.cloud.stream.app:throughput-sink-kafka:@registry.app.defaultVersion@
 sink.websocket=maven://org.springframework.cloud.stream.app:websocket-sink-kafka:@registry.app.defaultVersion@
 task.timestamp=maven://org.springframework.cloud.task.app:timestamp-task:@registry.app.defaultVersion@
```

Comment:
I updated the PR with what Ilaya had requested.  Keep in mind that if a user wants to treat a task as a source then they will need to add it to {binder}-apps.properties.  But with the starter task apps here they will not be dispatching messages to downstream apps.  If we do change this then we can move them back to the {binder}-apps.properties.